network/bridge/vars.go

```diff
@@ -0,0 +1,18 @@
+package bridge
+
+const (
+	// DefaultBridge defines the default bridge name.
+	DefaultBridge = "p0"
```

Comment:
"pouch0" is better?

Reply:
I agree, pouch0 is better.

Reply:
I think 'p0' is simple, different from 'xxx0'